examples/async/async.metatags

```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates the usage of async functionality in Ballerina, a way to execute operations asynchronously by returning a `future`.
+keywords: ballerina, ballerina-lang, ballerina by example, bbe, async, future, wait
```

Comment:
```suggestion
keywords: ballerina, ballerina by example, bbe, async, asynchronous, future, wait```

Reply:
Since we have already optimized for Ballerinalang, we don't have to add it here. You can remove that from other BBEs as well